sfm_pc/management/commands/download_data.py

```diff
@@ -0,0 +1,222 @@
+import os
+import csv
+import io
+
+from oauth2client.service_account import ServiceAccountCredentials
+from apiclient.discovery import build
+from googleapiclient.http import MediaIoBaseDownload
+import httplib2
+
+from tqdm import tqdm
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = 'Download entity, source, and location data from Google'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--sources_doc_id',
+            dest='sources_doc_id',
+            default='1o-K13Od1pGc7FOQ-JS8LJW9Angk3_UPZHTmOU7hD3wU',
+            help='Import data from specified Google Sheets Document'
+        )
+        
+        parser.add_argument(
+            '--location_doc_id',
+            dest='location_doc_id',
+            help='Import location data from specified Google Drive file'
+        )
+
+        parser.add_argument(
+            '--entity_doc_id',
+            dest='entity_doc_id',
+            default='1o-K13Od1pGc7FOQ-JS8LJW9Angk3_UPZHTmOU7hD3wU',
+            help='Import data from specified Google Sheets Document'
+        )
+        
+        parser.add_argument(
+            '--country_code',
+            dest='country_code',
+            help='Country code for the import'
+        )
+
+    def handle(self, *args, **kwargs):
+        entity_doc_id = kwargs['entity_doc_id']
+        location_doc_id = kwargs['location_doc_id']
+        sources_doc_id = kwargs['sources_doc_id']
+        country_code = kwargs['country_code'].rstrip()
+        
+        parent_directory = f'data'
+        country_directory = f'{parent_directory}/countries/{country_code}'
```

Comment:
It's a little confusing to define these here and pass them around. What do you think about storing the output directory and country code as class attributes, then creating the filepaths in the methods that actually create the files?

Reply:
see comment above